### PR TITLE
add support for kali linux rolling releases

### PR DESCRIPTION
**Pull request by azet**
Kali recently switched to a rolling release model: https://www.kali.org/news/kali-linux-rolling-edition-2016-1/

(CLA signed, added myself to `CONTRIBUTORS`)

### CODE REVIEW

File: installer/kali/releases [hunk 1]
```diff
@@ -1,2 +1,3 @@
 kali*
 sana
```
Comment (divx118) [inline installer/kali/releases]: I think you should also make sana unsupported like kali already is. 

> Please note that the Kali sana repositories will no longer be updated and will be EOL’d on the 15th of April 2016.

Comment (azet) [inline installer/kali/releases]: I wasn't sure about that as it's still supported. I will add a commit disabling sana.

**Comment by dnschneid** [comment]
Thanks!

**Comment by tonyxue** [comment]
Shouldn't the ```CONTRIBUTORS``` be alphabetical?

**Comment by dnschneid** [comment]
It's actually auto-generated by the Makefile every time I go to release, so I didn't bother nitting it here :)

**Pull request merged**